elements/pfe-sass/maps/_colors.scss

```diff
@@ -119,6 +119,7 @@ $pfe-colors: (
 
   // Palette of Border Colors
   surface--border--lightest:          $pf-color-black-150, // #f5f5f5
+  surface--border--lighter:           $pf-color-black-200, // #f0f0f0
```

Comment:
this color has been deprecated. the surface-base color `--theme--color--surface--base` is #f0f0f0

Reply:
Bump ^